Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/PANOSPolicyOptimizer.yml

```diff
@@ -47,6 +47,9 @@ script:
       predefined:
       - pre
       - post
+    - description: Is there an alternate device group to query?
+      name: override_device_groupoverride_device_group
```

Comment:
I believe this is a typo.

Reply:
Enitrely correct, should be addressed in latest commit